README.md

```diff
@@ -56,6 +56,7 @@ A curated list of awesome things and projects built by Algerian developers.
   - [Facial-expression-recognition-svm](https://github.com/amineHorseman/facial-expression-recognition-svm) - Training SVM classifier to recognize facial expressions (emotions)
   - [Malware Revealer](https://github.com/malware-revealer/malware-revealer) - A malware classification framework, designed primarily for malware detection, it contains a modular toolset for feature extraction, as well as pre-trained models and a ready to use web API for making predictions.
   - [Tchamba.random](https://github.com/Fcmam5/tchamba) package of functions generating random data (colors, names, jokes, letters..)
+  - [Lockdoor Framework ](https://github.com/SofianeHamlaoui/Lockdoor-Framework) -  A Penetration Testing framework with Cyber Security Resources
```

Comment:
Can we put it just before "Malware Revealer" (alphabetical order)? Thanks

Reply:
As you like, I just thought that we had to add them by (date of commit).
But sure, why not 😄

Reply:
I'm actually waiting for the reordering before merning, as you can see, everythinig is already alphabetically ordered in the file (per section)

Reply:
Aw perfect, I'll do a new PR after reordering them